src/store/concepts/lists/__tests__/actions.spec.js

```diff
@@ -0,0 +1,109 @@
+import {
+  GET_LISTS,
+  SAVE_LISTS,
+  SAVE_LISTS_TOTAL,
+  ADD_LIST,
+  LOAD_LISTS,
+  DELETE_LIST
+} from '../types'
+import {
+  getLists,
+  saveListsIds,
+  saveTotal,
+  addList,
+  loadLists,
+  deleteList
+} from '../actions'
+
+describe('lists actions', () => {
+  describe('loadLists()', () => {
+    describe('with default page', () => {
+      it('returns object with correct shape', () => {
+        expect(loadLists())
+          .toStrictEqual({
+            type: LOAD_LISTS,
+            page: 1
+          })
+      })
+    })
+
+    describe('with page', () => {
+      it('returns object with correct shape', () => {
+        expect(loadLists(5))
+          .toStrictEqual({
+            type: LOAD_LISTS,
+            page: 5
+          })
+      })
+    })
+  })
+
+  describe('getLists()', () => {
+    describe('with default page', () => {
```

Comment:
did you mean without page?

Reply:
I remove this test and change functional